modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsSLA.testcase

```diff
@@ -1108,6 +1100,10 @@ definition {
 	test SLASettingsAvailableOnMetricsPage {
 		WorkflowMetrics.goToWorkflowProcessMetrics(workflowProcessName = "Single Approver");
 
+		// Workaround for LPS-186021
```

Comment:
I didn't add the goToWorkflowSLASettings macro here because we don't need all of its steps in this test step